frontend/test/metabase/scenarios/admin/people/people.cy.spec.js

```diff
@@ -187,6 +183,78 @@ describe("scenarios > admin > people", () => {
       );
       cy.findByText(/^temporary password$/i).should("not.exist");
     });
+
+    it("should allow to search people", () => {
+      cy.visit("/admin/people");
+
+      cy.findByPlaceholderText("Find someone").type("no");
+      cy.findByText("5 people found");
+      assertTableRowsCount(5);
+
+      cy.findByPlaceholderText("Find someone").type("ne");
+      cy.findByText("1 people found");
+      assertTableRowsCount(1);
+
+      cy.findByPlaceholderText("Find someone").clear();
+      cy.findByText("8 people found");
+      assertTableRowsCount(8);
+    });
+
+    describe("pagination", () => {
+      beforeEach(() => {
+        generateUsers(8);
+      });
+
+      it("should allow paginating people forward and backward", () => {
+        cy.visit("/admin/people");
+
+        // Total
+        cy.findByText("16 people found");
+
+        // Page 1
+        cy.findByText("1 - 10");
+        assertTableRowsCount(10);
+        cy.findByTestId("previous-page-btn").should("be.disabled");
+
+        cy.findByTestId("next-page-btn").click();
+
+        // Page 2
+        cy.findByText("11 - 16");
+        assertTableRowsCount(6);
+        cy.findByTestId("next-page-btn").should("be.disabled");
+
+        cy.findByTestId("previous-page-btn").click();
+
+        // Page 1
+        cy.findByText("1 - 10");
+        assertTableRowsCount(10);
+      });
+
+      it("should allow paginating group members forward and backward", () => {
+        cy.visit("admin/people/groups/1");
+
+        // Total
+        cy.findByText("16 members");
+
+        // Page 1
+        cy.findByText("1 - 15");
+        assertTableRowsCount(15);
+        cy.findByTestId("previous-page-btn").should("be.disabled");
```

Comment:
You can alias this and then later use it in the test:
```suggestion
        cy.findByTestId("previous-page-btn")
          .as("prev")
          .should("be.disabled");
```

and to use it later:
`cy.get("@prev").click();`

Reply:
If you don't have objections I'd prefer to leave these as `previous-page-btn` because `prev` is a bit less descriptive and can mean the previous page

Reply:
Ofc, do whatever makes more sense to you.
This comment had more to do with the aliasing than what exact string will you use. In a small test like this it doesn't make much difference, but for longer tests it might be easier later to do `cy.get("@alias");`